intake/catalog/base.py

```diff
@@ -58,6 +57,8 @@ def __init__(self, entries=None, name=None, description=None, metadata=None,
         ttl : float, optional
             Lifespan (time to live) of cached modification time. Units are in
             seconds. Defaults to 1.
+        auth : BaseClientAuth or None
```

Comment:
This should be moved up to be above ttl?

Reply:
Oops, yes.